cmake/Modules/FindQwt.cmake

```diff
@@ -51,13 +51,13 @@ if(QWT_INCLUDE_DIRS)
   set(QWT_VERSION "No Version")
   string(REGEX MATCH "[0-9]+.[0-9]+.[0-9]+" QWT_VERSION ${QWT_STRING_VERSION})
   string(COMPARE LESS ${QWT_VERSION} "5.2.0" QWT_WRONG_VERSION)
-  string(COMPARE GREATER ${QWT_VERSION} "6.2.0" QWT_WRONG_VERSION)
+  string(COMPARE GREATER ${QWT_VERSION} "6.3.0" QWT_WRONG_VERSION)
 
   message(STATUS "QWT Version: ${QWT_VERSION}")
   if(NOT QWT_WRONG_VERSION)
     set(QWT_FOUND TRUE)
   else(NOT QWT_WRONG_VERSION)
-    message(STATUS "QWT Version must be >= 5.2 and <= 6.2.0, Found ${QWT_VERSION}")
+    message(STATUS "QWT Version must be >= 5.2 and <= 6.3.0, Found ${QWT_VERSION}")
```

Comment:
`string(COMPARE GREATER_EQUAL ${QWT_VERSION} "6.3.0" QWT_WRONG_VERSION)`
`message(STATUS "QWT Version must be >= 5.2 and < 6.3.0, Found ${QWT_VERSION}")`